packages/plugin-core/src/commands/pods/ConfigureServiceConnection.ts

```diff
@@ -12,50 +9,43 @@ import { BasicCommand } from "../base";
 
 type CommandOutput = void;
 
-type CommandInput = { serviceType: ExternalService };
-
-type CommandOpts = CommandInput;
+type CommandOpts = {};
 
 export class ConfigureServiceConnection extends BasicCommand<
   CommandOpts,
   CommandOutput
 > {
   key = DENDRON_COMMANDS.CONFIGURE_SERVICE_CONNECTION.key;
 
-  async gatherInputs(): Promise<CommandInput | undefined> {
-    const serviceType = await PodUIControls.promptForExternalServiceType();
-    if (!serviceType) return;
-    return {
-      serviceType,
-    };
-  }
-
-  async execute(opts: CommandOpts) {
+  async execute(_opts: CommandOpts) {
     const ctx = { ctx: "ConfigureServiceConnection" };
-    const { serviceType } = opts;
-    this.L.info({ ctx, serviceType, msg: "enter" });
+    this.L.info({ ctx, msg: "enter" });
     let configFilePath: string;
     const mngr = new ExternalConnectionManager(getExtension().podsDir);
-    const existingConnections = await mngr.getAllConfigsByType(serviceType);
-    if (existingConnections.length > 0) {
-      const items = existingConnections.map<QuickPickItem>((value) => {
-        return { label: value.connectionId };
-      });
-      const userChoice = await window.showQuickPick(items, {
+    const allServiceConfigs = await mngr.getAllValidConfigs();
+    const items = allServiceConfigs.map<QuickPickItem>((value) => {
+      return { label: value.connectionId };
```

Comment:
Can you also add `detail: value.serviceType` in the quickpick items?

Reply:
added service type in description.